### PR TITLE
feat(Model): Added batchnormalization to encoder output.

### PR DESCRIPTION
**Pull request by szemyd**

### CODE REVIEW

File: single_epoch.py [hunk 1]
```diff
@@ -7,14 +7,22 @@
...
+from tqdm.autonotebook import tqdm
 from sklearn.metrics import roc_auc_score
 from torch_geometric.loader import NeighborLoader, LinkNeighborLoader
 from utils.metrics_encoder_decoder import get_metrics_universal
 from utils.get_info import select_properties
+from utils.python_type import isnotebook
 
 device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
 
+# if isnotebook():
```
Comment (itchingpixels) [inline single_epoch.py:19]: is this then not necessary?

File: utils/python_type.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import os
+
+
+def isnotebook():
```
Comment (itchingpixels) [inline utils/python_type.py:4]: this one either, right?

**Review by itchingpixels — APPROVED** [review]
👍

File: single_epoch.py (hunk 1 above)
Comment (szemyd) [reply single_epoch.py:19]: Yes, you are right. We do either or. To be honest the original version gave more flexibility as we were checking for systems manually.

File: utils/python_type.py (hunk 2 above)
Comment (szemyd) [reply utils/python_type.py:4]: I'm going to take out the rest.
